services/src/main/java/org/keycloak/protocol/oidc/utils/AcrUtils.java

```diff
@@ -0,0 +1,88 @@
+package org.keycloak.protocol.oidc.utils;
```

Comment:
It might be good if you could add header comments the same as other newly added source files like AuthenticatorUtil.java .